### ISSUE TITLE
Images sent from desktop to mobile are cropped

### ISSUE THREAD

**Issue by qoqobolo** [status-im/status-desktop#10944]
[Original mobile issue](https://github.com/status-im/status-mobile/issues/16048)
Also see [this comment](https://github.com/status-im/status-mobile/issues/16048#issuecomment-1574904003) 

## Problem 
Any images, sent from desktop are cropped and resized.

#### Expected behavior

can see the whole picture

#### Actual behavior

https://github.com/status-im/status-mobile/assets/4557972/81245728-a157-4ef2-a38c-d57c20e047a6

### Reproduction

1) Sent any imge from desktop and try to open it on mobile device

### Additional Information

- Status version: RC 10, 1.23
- Desktop version: 0.12.0 
- Operating System: Android, iOS

**Comment by caybro** [comment]
@qoqobolo this could get fixed in recent master; can you pls try again with a build from https://github.com/status-im/status-desktop/pull/11004?

**Comment by anastasiyaig** [comment]
i tested it with master and it seems to be not yet fixed @caybro . Reproduced as described

**Comment by qoqobolo** [comment]
@caybro tried it in today's master (thanks @anastasiyaig for the assistance), the issue is still reproduced :(

**Comment by caybro** [comment]
Ok then this issue stays open for now ☺️

**Comment by kounkou** [comment]
Hi,

I did some testings about the issue. To do so : 
I install 1.20.4 of status-mobile (link below)
https://github.com/status-im/status-mobile/releases/tag/1.20.4
After what I used Genymotion to be able to emulate an android device 
`Google 12.0 - API 31 768x1280 320 - XHDPI 1.88 MB Genymotion (source)`
Then I installed the Status application.
I had to change the fleet to be able to communicate between Desktop app and mobile app
Once done, I sent a few images, here is a screenshot

![Screenshot from 2023-12-20 16-27-26](https://github.com/status-im/status-desktop/assets/2589171/67d226ce-45de-49d6-ae18-397853e98ea2)

I opened the images to observe if the image characteristics matches the issue

![Screenshot from 2023-12-20 16-29-45](https://github.com/status-im/status-desktop/assets/2589171/a16a0d8e-228f-407c-8d9e-da8d68761e33)

From above observations, the issue doesn't exist anymore.

I'll close the issue
